### PR TITLE
List AltEstate (ALT)

### PR DESCRIPTION
**Pull request by ctapu4ok**
- Official project URL: https://alt.estate/
- Official block explorer URL: https://etherscan.io/token/0x419b8ed155180a8c9c64145e76dad49c0a4efb97

### CODE REVIEW

File: assets/src/test/java/bisq/asset/tokens/AltEstateTokenTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+package bisq.asset.tokens;
+
+import bisq.asset.AbstractAssetTest;
+
+import org.junit.Test;
+
+public class AltEstateTokenTest extends AbstractAssetTest {
+
+    public AltEstateTokenTest() {
+        super(new Exceed());
```
Comment (blabno) [inline assets/src/test/java/bisq/asset/tokens/AltEstateTokenTest.java:27]: Wrong token

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

File: assets/src/test/java/bisq/asset/tokens/AltEstateTokenTest.java (hunk 1 above)
Comment (ctapu4ok) [reply assets/src/test/java/bisq/asset/tokens/AltEstateTokenTest.java:27]: Why the wrong token?

Comment (blabno) [reply assets/src/test/java/bisq/asset/tokens/AltEstateTokenTest.java:27]: Shouldn't it be `super(new AltEstateToken())`?

**Comment by ManfredKarrer** [comment]
Closed as not updated after 2 weeks. Open again after it is fixed.

**Pull request closed without merging**